python_backend/api.py

```
@@ -856,6 +865,942 @@ async def generate_slide_previews_endpoint(
+        pdf_file.write_bytes(pdf_bytes)
+
+        from PIL import Image
+
+        with fitz.open(pdf_file) as doc:
+            processed_count = 0
+
+            for page_idx in todo_pages:
+                # タイムアウト耐性: 残り時間チェック
+                elapsed = time.time() - start_time
+                if elapsed > REQUEST_TIMEOUT_SECONDS:
+                    # 安全に抜ける（次回呼び出しで続きを処理）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)